components/VerifyOptions.yml

```diff
@@ -23,9 +23,6 @@ components:
         proofPurpose:
           type: string
```

Comment:
Hmm is it really a hard rule that for VC the proofPurpose must be "assertionMethod", and for VP it must be "authentication", or are there legitimate cases where this is could be different?

Reply:
+1 to making it an enum. (Because I agree, a case could be made that the VP proof purpose _can_ be `assertionMethod`.)

Reply:
Anyone can also come up with additional proof purposes, i.e. this is completely extensible, no?

Reply:
I don't think we should be adding optionality without use cases. I don't know of any use cases for a VC that is signed for a purpose other than making an assertion nor do I know of any VP that is signed for a purpose other than authentication. If those exist, they should be listed as inputs for this API -- if they don't, we shouldn't offer other options in the proposed standard so that we can keep things simple.

Reply:
That makes sense to me too, thanks for the explanation. Anyway, this thread seems orthogonal to the PR and should probably be discussed in a separate issue.